src/app.ts

```diff
@@ -28,6 +29,7 @@ class App {
 
 		this._middleware();
 		this._routes();
+		setErrorHandling(this.express);
```

Comment:
move this call to the middleware function as the function internally sets the error handling as middleware.

Reply:
They have to be in the right order... If we want to be cleaner, we can put the initialization in the _routes? Do you agree with that?